### DISCUSSION TITLE
Enable the various character service commands on the PTR.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#5866] [Suggestions]
Things like race change, faction change, customize, etc. People ask questions about what all they entail when they could just copy their character onto the PTR, run a command, and see for themself what it would look like.

**Comment by heyitsbench** [comment]
The commands as far as I know them are as follows:
```
.character changefaction - Change Faction
.character changerace    - Change Race
.character customize     - Character Customize
```

**Comment by Vyperhand** [comment]
I would further say that far more detailed descriptions of the exact function provided being added to the store entries are in order as well.  For example, the Race Change had some unexpected behavior that would definitely have made me wait just a little bit before buying one.

The character customization picture in particular is aggressively misleading, showing customizations that can't be chosen with the service.
